doc/ug/documentation.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Building documentation"
+---
+
+The documentation for OpenTitan is available [online](https://docs.opentitan.org).
+The creation of documentation is mainly based around the conversion from Markdown to HTML files with [Hugo](https://gohugo.io/).
+Rules for how to write correct Markdown files can be found in the [reference manual]({{< relref "doc/rm/markdown_usage_style.md" >}}).
+
+## Building locally
+
+Before Hugo is executed a few project specific processing steps are necessary.
+These steps require the installation of the dependencies outlined in the following.
+All processing steps as well as the invocation to Hugo are combined in the script `./util/build_docs.py`.
```

Comment:
maybe remove the `./` preceding `util/build_docs.py` since that implies it is until `doc/ug`?

Reply:
Thanks. Removed the prefix.